doc/classes/Array.xml

```diff
@@ -669,6 +669,14 @@
 				[b]Note:[/b] The [param position] cannot be negative. To remove an element relative to the end of the array, use [code]arr.remove_at(arr.size() - (i + 1))[/code]. To remove the last element from the array, use [code]arr.resize(arr.size() - 1)[/code].
 			</description>
 		</method>
+		<method name="reserve">
+			<return type="int" />
+			<param index="0" name="min_capacity" type="int" />
+			<description>
+				Reserve space for at least [param min_capacity] total elements.
```

Comment:
Seconding what was said in the discussion, I think it's important to mention what exactly this does and the benefits, like "if a certain amount of capacity is reserved, subsequent `push_backs` will be sped up until the size gets bigger than that".

Reply:
Thanks for the feedback :)
We've reached a conclusion in the last core meeting to _not_ expose this function to GDScript for now, as it's not that useful in comparison (and potentially confusing against `resize`). So my next push will remove this change anyway!